src/main/java/uk/sky/cqlmigrate/TableChecker.java

```diff
@@ -0,0 +1,51 @@
+package uk.sky.cqlmigrate;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import org.awaitility.Awaitility;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+import static java.util.Objects.nonNull;
+
+public class TableChecker {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TableChecker.class);
+    public static final String AWSK_CLUSTER_NAME = "Amazon Keyspaces";
+    public static final String TABLE_ACTIVE_STATUS = "ACTIVE";
+
+
+    public static void check(CqlSession session, String keyspace) {
+        String tableUpdateTimeoutString = System.getProperty("tableUpdateTimeout");
```

Comment:
I don't believe we can use system properties as cqlmigrate may be used as a library inside an app or as a standalone runnable, like we do in cqlmigrator.
When used inside an app, this system property likely won't be set and we shouldn't rely on the app to set it. Instead it should be passed in when constructing the CqlMigrate instance. Likely in something like the `CqlMigratorConfig` object

Reply:
Correct, I should have read docs more carefully. But when we have CqlMigratorConfig why are almost all configuration just passed through as parameters? I wanted to avoid extending list of parameters in many methods.